E-Commerce/readme.md

```diff
@@ -46,13 +46,101 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+
+SELECT * 
+FROM products
+WHERE product_name LIKE '%socks%';
+
+
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+ SELECT p.id, p.product_name, pa.unit_price, pa.supp_id
+ FROM products AS p
+ INNER JOIN product_availability AS pa
+ ON p.id = pa.prod_id
+ WHERE pa.unit_price > 100;
+
+
+
+
 - [ ] List the 5 most expensive products
+
```

Comment:
Here in your results, there are actually only two products (Mobile Phone X and Javascript Book), because there are multiple suppliers for each product. One way to interpret the question could be to instead look at the most expensive supplier for each product and return the top 5 from that. You could try that as an extra challenge.